ports/spix/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "spix",
+  "version": "0.2",
+  "description": "A minimally invasive UI testing library that enables your Qt/QML app's UI to be controlled either via c++ code, or through an http RPC interface.",
+  "homepage": "https://github.com/faaxm/spix",
+  "dependencies": [
+    "anyrpc",
+    {
+      "name": "qt5-base",
+      "default-features": false
+    },
+    "qt5-declarative"
```

```suggestion
    "qt5-declarative",
      {
      "name": "vcpkg-cmake",
      "host": true
     }
```
Please add the dependencies to format vcpkg.json, then rebuild this.

Sorry, I missed this point before.

---

I hope this is what you asked for https://github.com/microsoft/vcpkg/pull/19351/commits/62cee16e2263b4e99f6d2aea4609c0a6665f5734